pkgs/development/python-modules/drf-nested-routers/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+, django
+, djangorestframework
+}:
+
+buildPythonPackage rec {
+  pname = "drf-nested-routers";
+  version = "0.92.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0rizimr0sxzspc8gn1fd2z1bp62x0wkkr29vwr4nbcg96dik4slx";
+  };
+
+  propagateBuildInputs = [ setuptools ];
+  buildInputs = [ django djangorestframework ];
```

Comment:
these should probably be added to `propagatedBuildInputs` if they are python packages

Reply:
I just noticed I was using `propagateBuildInputs` instead of `propagatedBuildInputs`.
Once I switched to the later one I was able to pass `django` as well.